hathor/merged_mining/coordinator.py

```diff
@@ -215,7 +224,7 @@ def _make_bitcoin_block_and_coinbase(self, work: SingleMinerWork) -> Tuple[Bitco
         )
         return bitcoin_header, coinbase_tx
 
-    def build_bitcoin_block_header(self, work: SingleMinerWork) -> BitcoinBlockHeader:
+    def build_parent_block_header(self, work: SingleMinerWork) -> BitcoinBlockHeader:
```

Comment:
This builds both BTC and DGB block headers?

Reply:
Yes. The header is exactly the same format.

Reply:
Thanks. Could you just add that to the docstring please? Or change the method name to use bitcoin again and we can add a comment when we call it from the DGB code to say the header is the same as BTC.

Reply:
Since I'm using "bitcoin like" terminology (in `BitcoinLikeAddress` that is used for DigiByte too), what do you think about `build_bitcoin_like_header`?

Reply:
Sounds good

Reply:
Done.